src/lv_widgets/lv_chart.h

```diff
@@ -122,7 +81,88 @@ enum {
  * from it
  * @return pointer to the created chart background
  */
-lv_obj_t * lv_chart_create(lv_obj_t * par, const lv_obj_t * copy);
+lv_obj_t * lv_chart_create(lv_obj_t * par, const lv_obj_t * copy)
```

Comment:
It seems somehow the implementation of the functions are moved to the header file.

Reply:
fixed. 
thanks